### ISSUE TITLE
[install failed] UnicodeDecodeError: 'gbk' codec can't decode byte 0xad in position 1843: illegal multibyte sequence

### ISSUE THREAD

**Issue by konosubakonoakua** [Trusted-AI/adversarial-robustness-toolbox#771]
**Describe the bug**
manual installation ending with failure due to “UnicodeDecodeError”

**To Reproduce**
1. Go to `root`
2. conda activate ART
3. pip install .

**Expected behavior**
get UnicodeDecodeError when installing on WIN

**Screenshots**
![image](https://user-images.githubusercontent.com/42881610/100584873-a92d6e00-3327-11eb-9b8f-b22f602873fd.png)
```shell
❯ pip install .
Looking in indexes: https://mirrors.bfsu.edu.cn/pypi/web/simple
Processing c:\users\13567\desktop\adversarial-robustness-toolbox
  Installing build dependencies ... done
  Getting requirements to build wheel ... error
  ERROR: Command errored out with exit status 1:
   command: 'D:\app\miniconda3\envs\ART\python.exe' 'D:\app\miniconda3\envs\ART\lib\site-packages\pip\_vendor\pep517\_in_process.py' get_requires_for_build_wheel 'C:\Users\13567\AppData\Local\Temp\tmpp4n8__k_'
       cwd: C:\Users\13567\AppData\Local\Temp\pip-req-build-1ysge9lo
  Complete output (18 lines):
  Traceback (most recent call last):
    File "D:\app\miniconda3\envs\ART\lib\site-packages\pip\_vendor\pep517\_in_process.py", line 280, in <module>
      main()
    File "D:\app\miniconda3\envs\ART\lib\site-packages\pip\_vendor\pep517\_in_process.py", line 263, in main
      json_out['return_val'] = hook(**hook_input['kwargs'])
    File "D:\app\miniconda3\envs\ART\lib\site-packages\pip\_vendor\pep517\_in_process.py", line 114, in get_requires_for_build_wheel
      return hook(config_settings)
    File "C:\Users\13567\AppData\Local\Temp\pip-build-env-1v9fljtq\overlay\Lib\site-packages\setuptools\build_meta.py", line 150, in get_requires_for_build_wheel
      config_settings, requirements=['wheel'])
    File "C:\Users\13567\AppData\Local\Temp\pip-build-env-1v9fljtq\overlay\Lib\site-packages\setuptools\build_meta.py", line 130, in _get_build_requires
      self.run_setup()
    File "C:\Users\13567\AppData\Local\Temp\pip-build-env-1v9fljtq\overlay\Lib\site-packages\setuptools\build_meta.py", line 254, in run_setup
      self).run_setup(setup_script=setup_script)
    File "C:\Users\13567\AppData\Local\Temp\pip-build-env-1v9fljtq\overlay\Lib\site-packages\setuptools\build_meta.py", line 145, in run_setup
      exec(compile(code, __file__, 'exec'), locals())
    File "setup.py", line 7, in <module>
      long_description = fh.read()
  UnicodeDecodeError: 'gbk' codec can't decode byte 0xad in position 1843: illegal multibyte sequence
  ----------------------------------------
ERROR: Command errored out with exit status 1: 'D:\app\miniconda3\envs\ART\python.exe' 'D:\app\miniconda3\envs\ART\lib\site-packages\pip\_vendor\pep517\_in_process.py' get_requires_for_build_wheel 'C:\Users\13567\AppData\Local\Temp\tmpp4n8__k_' Check the logs for full command output.
```
**System information (please complete the following information):**
 - OS WIN10 20H2
 - Python version 3.6
 - ART version dev_1.5.0

**Comment by konosubakonoakua** [comment]
adding `encoding="utf-8"` in `open()` of `setup.py` solved this problem.
```shell
❯ pip install .
Looking in indexes: https://mirrors.bfsu.edu.cn/pypi/web/simple
Processing c:\users\13567\desktop\adversarial-robustness-toolbox
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
    Preparing wheel metadata ... done
Requirement already satisfied: cma in d:\app\miniconda3\envs\art\lib\site-packages (from adversarial-robustness-toolbox==1.4.3) (3.0.3)
Requirement already satisfied: mypy in d:\app\miniconda3\envs\art\lib\site-packages (from adversarial-robustness-toolbox==1.4.3) (0.790)
Requirement already satisfied: scipy>=1.4.1 in d:\app\miniconda3\envs\art\lib\site-packages (from adversarial-robustness-toolbox==1.4.3) (1.4.1)
Requirement already satisfied: six in d:\app\miniconda3\envs\art\lib\site-packages (from adversarial-robustness-toolbox==1.4.3) (1.15.0)
Requirement already satisfied: statsmodels in d:\app\miniconda3\envs\art\lib\site-packages (from adversarial-robustness-toolbox==1.4.3) (0.12.1)
Requirement already satisfied: pydub in d:\app\miniconda3\envs\art\lib\site-packages (from adversarial-robustness-toolbox==1.4.3) (0.24.1)
Requirement already satisfied: numpy in d:\app\miniconda3\envs\art\lib\site-packages (from adversarial-robustness-toolbox==1.4.3) (1.19.2)
Requirement already satisfied: tqdm in d:\app\miniconda3\envs\art\lib\site-packages (from adversarial-robustness-toolbox==1.4.3) (4.54.0)
Requirement already satisfied: Pillow in d:\app\miniconda3\envs\art\lib\site-packages (from adversarial-robustness-toolbox==1.4.3) (8.0.1)
Requirement already satisfied: resampy in d:\app\miniconda3\envs\art\lib\site-packages (from adversarial-robustness-toolbox==1.4.3) (0.2.2)
Requirement already satisfied: ffmpeg-python in d:\app\miniconda3\envs\art\lib\site-packages (from adversarial-robustness-toolbox==1.4.3) (0.2.0)
Requirement already satisfied: matplotlib in d:\app\miniconda3\envs\art\lib\site-packages (from adversarial-robustness-toolbox==1.4.3) (3.3.3)
Requirement already satisfied: setuptools in d:\app\miniconda3\envs\art\lib\site-packages (from adversarial-robustness-toolbox==1.4.3) (50.3.1.post20201107)
Requirement already satisfied: scikit-learn>=0.22.2 in d:\app\miniconda3\envs\art\lib\site-packages (from adversarial-robustness-toolbox==1.4.3) (0.22.2)
Requirement already satisfied: mypy-extensions<0.5.0,>=0.4.3 in d:\app\miniconda3\envs\art\lib\site-packages (from mypy->adversarial-robustness-toolbox==1.4.3) (0.4.3)
Requirement already satisfied: typing-extensions>=3.7.4 in d:\app\miniconda3\envs\art\lib\site-packages (from mypy->adversarial-robustness-toolbox==1.4.3) (3.7.4.3)
Requirement already satisfied: typed-ast<1.5.0,>=1.4.0 in d:\app\miniconda3\envs\art\lib\site-packages (from mypy->adversarial-robustness-toolbox==1.4.3) (1.4.1)
Requirement already satisfied: patsy>=0.5 in d:\app\miniconda3\envs\art\lib\site-packages (from statsmodels->adversarial-robustness-toolbox==1.4.3) (0.5.1)
Requirement already satisfied: pandas>=0.21 in d:\app\miniconda3\envs\art\lib\site-packages (from statsmodels->adversarial-robustness-toolbox==1.4.3) (1.1.4)
Requirement already satisfied: numba>=0.32 in d:\app\miniconda3\envs\art\lib\site-packages (from resampy->adversarial-robustness-toolbox==1.4.3) (0.51.2)
Requirement already satisfied: future in d:\app\miniconda3\envs\art\lib\site-packages (from ffmpeg-python->adversarial-robustness-toolbox==1.4.3) (0.18.2)
Requirement already satisfied: pyparsing!=2.0.4,!=2.1.2,!=2.1.6,>=2.0.3 in d:\app\miniconda3\envs\art\lib\site-packages (from matplotlib->adversarial-robustness-toolbox==1.4.3) (2.4.7)
Requirement already satisfied: cycler>=0.10 in d:\app\miniconda3\envs\art\lib\site-packages (from matplotlib->adversarial-robustness-toolbox==1.4.3) (0.10.0)
Requirement already satisfied: kiwisolver>=1.0.1 in d:\app\miniconda3\envs\art\lib\site-packages (from matplotlib->adversarial-robustness-toolbox==1.4.3) (1.3.1)
Requirement already satisfied: python-dateutil>=2.1 in d:\app\miniconda3\envs\art\lib\site-packages (from matplotlib->adversarial-robustness-toolbox==1.4.3) (2.8.1)
Requirement already satisfied: joblib>=0.11 in d:\app\miniconda3\envs\art\lib\site-packages (from scikit-learn>=0.22.2->adversarial-robustness-toolbox==1.4.3) (0.17.0)
Requirement already satisfied: pytz>=2017.2 in d:\app\miniconda3\envs\art\lib\site-packages (from pandas>=0.21->statsmodels->adversarial-robustness-toolbox==1.4.3) (2020.4)
Requirement already satisfied: llvmlite<0.35,>=0.34.0.dev0 in d:\app\miniconda3\envs\art\lib\site-packages (from numba>=0.32->resampy->adversarial-robustness-toolbox==1.4.3) (0.34.0)
Building wheels for collected packages: adversarial-robustness-toolbox
  Building wheel for adversarial-robustness-toolbox (PEP 517) ... done
  Created wheel for adversarial-robustness-toolbox: filename=adversarial_robustness_toolbox-1.4.3-py3-none-any.whl size=884041 sha256=62842fcd8e78d9fd9a65ec4a0c6fe49b6be1ee9282fe8020da13e9ab4a6e5293
  Stored in directory: c:\users\13567\appdata\local\pip\cache\wheels\36\c9\e5\90853a39eaf5ff136ee59be611cdba50d8c1ce6b7095746bc1
Successfully built adversarial-robustness-toolbox
Installing collected packages: adversarial-robustness-toolbox
Successfully installed adversarial-robustness-toolbox-1.4.3
```
**Screenshot**
![image](https://user-images.githubusercontent.com/42881610/100585689-ed6d3e00-3328-11eb-9827-d7c8203c3bd2.png)

**Comment by beat-buesser** [comment]
Hi @konosubakonoakua Thank you very much for raising this issue and providing the tested solution! I have included the definition of the encoding in #772 and it will released as part of ART 1.5.0 later today.
